### PR TITLE
version check script for ci

### PR DESCRIPTION
**Pull request by f1337**
See https://app.zenhub.com/workspaces/vsp-5cedc9cce6e3335dc5a49fc4/issues/department-of-veterans-affairs/va.gov-team/12412 for details.

### CODE REVIEW

File: dist/10-10CG-example.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "veteran": {
     "fullName": {
-      "first": "Jane Doe",
+      "first": "Jan Doe",
```
Comment (f1337) [inline dist/10-10CG-example.json:4]: This is a test change, to trigger a CI failure. It will not be merged.

**Comment by omgitsbillryan** [comment]
So any changes to `.json` files or any files in `/src/schema` will result in a circleci failure - am I reading this properly?

I'm a `vets-json-schema` rookie, but looking at the past few commits to this repo it seems like normal workflow to change these files. I think I'm missing something.

File: bin/version_check [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#! /usr/bin/env bash
+
+# USAGE: ./bin/version_check || circleci step halt
+
+changes=`git show -m HEAD --name-only --pretty=""`
```
Comment (omgitsbillryan) [inline bin/version_check:5]: I need to be reminded - what does `git show` show? Is it only the last commit? What if a PR has multiple commits?

Comment (f1337) [reply bin/version_check:5]: > So any changes to `.json` files or any files in `/src/schema` will result in a circleci failure - am I reading this properly?

Right, this is just the first step, making sure the bash script can fail CircleCI.

Next step is to only fail if there are those changes and package.json isn't also changes.

Then I want to actually check that the version string of package.json is updated.

The stretch goal would be to check that the version string is *incremented* in package.json. But I think the tool is still useful if we stop at the immediate previous step.

**Comment by f1337** [comment]
> So any changes to `.json` files or any files in `/src/schema` will result in a circleci failure - am I reading this properly?

Right, this is just the first step, making sure the bash script can fail CircleCI.

Next step is to only fail if there are those changes and package.json isn't also changes.

Then I want to actually check that the version string of package.json is updated.

The stretch goal would be to check that the version string is *incremented* in package.json. But I think the tool is still useful if we stop at the immediate previous step.

File: bin/version_check (hunk 2 above)
Comment (f1337) [reply bin/version_check:5]: Yeah, it'll be `git diff origin/master` in the next iteration.

**Pull request closed without merging**